### PR TITLE
Machine Learning

### PR DESCRIPTION
**Pull request by Franco98404**
- Improvements in the exceptions
- Code more distributed
- Docker incorporation
- Minor changes

### CODE REVIEW

File: MACHINE_LEARNING/image_recognizer/image_recognizer_app/model/yolo.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 import numpy as np
 from .model import Model
 from pathlib import Path
+from ..exceptions.yolo_exception import YoloException
 
 
 class Yolo(Model):
```
Comment (lcarrietas) [inline MACHINE_LEARNING/image_recognizer/image_recognizer_app/model/yolo.py:21]: Please add Yolo class description

File: MACHINE_LEARNING/image_recognizer/image_recognizer_app/views.py [hunk 2]
```diff
@@ -10,29 +10,36 @@
...
 from pathlib import Path
 from .model.prediction import Prediction
 from .utils.checker import Checker
 from .utils.unzip import Unzip
-
+from django.http import HttpResponse
+from .utils.imageRecognizer import ImageRecognizer
+import json
+from .exceptions.machine_learning_exception import MachineLearningException
+from .exceptions.error_response import ErrorResponse
```
Comment (awielern) [inline MACHINE_LEARNING/image_recognizer/image_recognizer_app/views.py:25]: Please add an space, to standardize the format in all the files

**Review by lcarrietas — COMMENTED** [review]
Please review comment

**Review by awielern — APPROVED** [review]
Looks good, just add that space and it's done.

File: MACHINE_LEARNING/image_recognizer/image_recognizer_app/model/yolo.py (hunk 1 above)
Comment (Franco98404) [reply MACHINE_LEARNING/image_recognizer/image_recognizer_app/model/yolo.py:21]: Done!

File: MACHINE_LEARNING/image_recognizer/image_recognizer_app/views.py (hunk 2 above)
Comment (Franco98404) [reply MACHINE_LEARNING/image_recognizer/image_recognizer_app/views.py:25]: Done!

**Review by lcarrietas — APPROVED** [review]
Nice!

**Pull request merged**